src/NathanThomas/NT-110.kt

```diff
@@ -0,0 +1,52 @@
+/*
+Given a non-negative int n, compute recursively (no loops)
+the count of the occurrences of 8 as a digit, except that
+an 8 with another 8 immediately to its left counts double,
+so 8818 yields 4. Note that mod (%) by 10 yields the rightmost
+digit (126 % 10 is 6), while divide (/) by 10 removes the
+rightmost digit (126 / 10 is 12).
+
+Examples:
+count8(8) → 1
+count8(818) → 2
+count8(8818) → 4
+*/
+
+package NathanThomas
+
+fun main(args: Array<String>) {
+
+    print("Enter any number: ")
+    val input = readLine()!!
+    val number: Int = input.toInt()
+
+    findEight(number, 0)
+}
+
+fun findEight(numbers: Int, previous: Int): Int {
```

Comment:
Correct me if I'm wrong, but I'm testing 110. My input was 18118818. From my understanding, the total should be 6 8's but it looks like your method is counting 5 8's.